config.go

```diff
@@ -3,83 +3,91 @@ package main
 import (
 	"time"
 
-	"github.com/BurntSushi/toml"
+	"github.com/kelseyhightower/envconfig"
 )
 
 type ListenerUrlsConfig struct {
-	Urls []string `toml:"urls"`
+	Urls []string `envconfig:"URLS"`
 }
 
 type HAproxyConfig struct {
-	ReloadCmd    string `toml:"reload_command"`
-	VerifyCmd    string `toml:"verify_command"`
-	BindIP       string `toml:"bind_ip"`
-	TemplateFile string `toml:"template_file"`
-	ConfigFile   string `toml:"config_file"`
-	PidFile      string `toml:"pid_file"`
-	Disable      bool   `toml:"disable"`
-	User         string `toml:"user"`
-	Group        string `toml:"group"`
-	UseHostnames bool   `toml:"use_hostnames"`
+	ReloadCmd    string `envconfig:"RELOAD_COMMAND"`
+	VerifyCmd    string `envconfig:"VERIFY_COMMAND"`
+	BindIP       string `envconfig:"BIND_IP" default:"192.168.168.168"`
+	TemplateFile string `envconfig:"TEMPLATE_FILE" default:"views/haproxy.cfg"`
+	ConfigFile   string `envconfig:"CONFIG_FILE" default:"/etc/haproxy.cfg"`
+	PidFile      string `envconfig:"PID_FILE" default:"/var/run/haproxy.pid"`
+	Disable      bool   `envconfig:"DISABLE"`
+	User         string `envconfig:"USER" default:"haproxy"`
+	Group        string `envconfig:"GROUP" default:"haproxy"`
+	UseHostnames bool   `envconfig:"USE_HOSTNAMES" default:"false"`
 }
 
 type ServicesConfig struct {
-	NameMatch    string `toml:"name_match"`
-	ServiceNamer string `toml:"namer"`
-	NameLabel    string `toml:"name_label"`
+	NameMatch    string `envconfig:"NAME_MATCH"`
+	ServiceNamer string `envconfig:"NAMER" default:"docker_label"`
```

Comment:
SERVICE_NAMER?

Reply:
The section is called `services` so if I named it that, it would be `SERVICES_SERVICE_NAMER` which seems a lot more confusing than `SERVICES_NAMER`